src/notebooks/controllers/condaControllerRefresher.node.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+'use strict';
+import { inject, injectable } from 'inversify';
+import { IExtensionSingleActivationService } from '../../platform/activation/types';
+import { IPythonExtensionChecker } from '../../platform/api/types';
+import { CondaService } from '../../platform/common/process/condaService.node';
+import { IDisposableRegistry } from '../../platform/common/types';
+import { waitForCondition } from '../../platform/common/utils/async';
+import { debounceAsync } from '../../platform/common/utils/decorators';
+import { IInterpreterService } from '../../platform/interpreter/contracts';
+import { EnvironmentType } from '../../platform/pythonEnvironments/info';
+import { INotebookControllerManager } from '../types';
+
+@injectable()
+export class CondaControllerRefresher implements IExtensionSingleActivationService {
```

Comment:
This file exists because the NotebookControllerManager used to listen to the conda service in order to refresh its list of controllers.

Conda Service is node only, so I created this one off to accomplish the same thing.